.github/workflows/fuzzer-indexing.yml

```diff
@@ -0,0 +1,24 @@
+name: Run the indexing fuzzer
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  fuzz:
+    name: Setup the action
+    runs-on: ubuntu-latest
+    timeout-minutes: 4320 # 72h
```

Comment:
Why do you need to put a time-out? How long should last the CI usually?

Reply:
I must admit that I overlooked that part a little bit :joy: 

While the fuzzer can't find any bugs, it runs forever.
I took the same timeout we had on the benchmark CI but:
- Now I remember that since the benchmarks run on our custom machine, it can run way longer than a normal CI
- Running for 3 days per commit doesn't really make sense

I don't know if it really makes sense to run the fuzzer on each commit, but I think we could run it for ~1h, maybe? :thinking: 
Or run it longer, but only when we make a release?